### DISCUSSION TITLE
Make it work on Live ISOs, too

### DISCUSSION THREAD

**Discussion by probonopd** [linuxdeepin/developer-center#2955] [Features Request & Ideas | 特性请求 & 头脑风暴]
Please make it work on Live ISOs, too

**Comment by BLumia** [comment]
It probably not possible to make deepin-graphics-driver-manager work on live ISO since it requires a reboot to make the system boot on overlayfs and verify if the driver can be installed on that system, and live ISO is supposed to be read-only and not be modified by any other software which include deepin-graphics-driver-manager.
